config/300-pubsub.yaml

```diff
@@ -65,6 +65,41 @@ spec:
             sink:
               type: object
               description: "Reference to an object that will resolve to a domain name to use as the sink."
+              anyOf:
+                - type: object
+                  properties:
+                    uri:
+                      type: string
+                      minLength: 1
+                - type: object
+                  properties:
```

Comment:
Should this list also include a uri field?

Reply:
Isn't that the first 'branch' of the anyOf?